src/Netflix/Netflix.ts

```diff
@@ -333,7 +373,7 @@ export default <IPluginBase & IPlugin>{
         normal: false
       },
       {
-        name: "ChangeText",
+        name: "Change Text",
         match: ["[change/switch] [text/subtitle] to *", "[text/subtitle] to *"],
```

Comment:
this can just be "[/change /switch ][text/subtitle] to *"

Reply:
Nice